src/query/Lambda.js

```diff
@@ -0,0 +1,50 @@
+import { InvalidValue } from '../errors'
+import Expr from '../Expr'
+import { checkInstanceHasProperty } from '../_util'
+import arity from './arity'
+import lambdaExpr from './lambdaExpr'
+import lambdaFunc from './lambdaFunc'
```

Comment:
is there any benefit of extracting `lambdaExpr` and `lambdaFunc` in separate files? I think those functions are only used here